package.json

```diff
@@ -1,83 +1,109 @@
 {
   "name": "react-boilerplate",
-  "version": "1.0.0",
-  "description": "Boilerplate for our React applications",
-  "main": "src/index.js",
-  "author": "Christoffer Niska <christofferniska@gmail.com>",
-  "license": "MIT",
+  "version": "2.0.0",
+  "private": true,
+  "dependencies": {
+    "normalize.css": "^6.0.0",
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "react-redux": "^5.0.4",
+    "react-router": "3",
+    "redux": "^3.6.0",
+    "redux-actions": "^2.0.2",
+    "redux-saga": "^0.14.7"
+  },
+  "devDependencies": {
```

Comment:
But why? We should be installing dev dependencies during CI and only leave them out when building production.